optimize/c4/.yarnrc.yml

```diff
@@ -1,3 +1 @@
 nodeLinker: node-modules
-
-yarnPath: .yarn/releases/yarn-4.2.2.cjs
```

Comment:
Hosting the yarn binary in the project folder is [not recommended](https://yarnpkg.com/configuration/yarnrc#yarnPath) anymore. Enabling coreback in node will correctly resolve the correct binary.